sdcm/sct_runner.py

```diff
@@ -61,22 +59,23 @@ def instance_root_disk_size(test_duration):
     @staticmethod
     @lru_cache(maxsize=None)
     def key_pair():
-        ks = KeyStore()
-        return ks.get_ec2_ssh_key_pair()
+        raise NotImplementedError()
 
     def get_remoter(self, host, connect_timeout: Optional[float] = None):
         self._ssh_pkey_file = tempfile.NamedTemporaryFile(mode="w", delete=False)
         self._ssh_pkey_file.write(self.key_pair().private_key.decode())
         self._ssh_pkey_file.flush()
         return RemoteCmdRunnerBase.create_remoter(hostname=host, user=self.LOGIN_USER, key_file=self._ssh_pkey_file.name, connect_timeout=connect_timeout)
 
-    def install_prereqs(self, public_ip):
-        LOGGER.info("Installing required packages...")
+    def install_prereqs(self, public_ip, connect_timeout=None):
         prereqs_script = dedent(f"""
             echo "fs.aio-max-nr = 65536" >> /etc/sysctl.conf
             echo "ubuntu soft nofile 4096" >> /etc/security/limits.conf
             echo "jenkins soft nofile 4096" >> /etc/security/limits.conf
             echo "root soft nofile 4096" >> /etc/security/limits.conf
+            mkdir -p /home/ubuntu/.ssh
```

Comment:
you run it under root, so you endup having .ssh with root privileges under /home/ubuntu, which is not nice
also if folde is there it will fail, so add "| true"

Reply:
it has '-p' option, the cmd won't fail even the directory exists.